hack/consecutive-upgrades-test.sh

```diff
@@ -0,0 +1,267 @@
+#!/bin/bash -e
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2023 Red Hat, Inc.
+#
+# Usage:
+# make upgrade-test-operator-sdk
+#
+# Use Openshift-CI "optional-operators-ci-*" workflow to:
+# - Use the operator-sdk to upgrade a pre deployed bundle; use a new bundle,
+#   named 100.0.0 with the contents of the open PR (this can include new
+#   dependent images, new CRDs...).
+# - the script then performs various validations against the upgraded version.
+
+MAX_STEPS=$(( $(grep -c "Msg " "$0") - 2)) # subtract self line and the function name
```

Comment:
We need to re-think about it. This is not true, as we're calling the `Msg` function for the `upgrade` function, so the number here is not right.

Let's fix this in a future PR.